applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -210,5 +338,7 @@ int bt_mgmt_init(void)
 		bt_mgmt_adv_init();
 	}
 
+	mac_print();
```

Comment:
I think this should be right after the k_sem_take(), just to have it more bundled together

Reply:
That looks better, but other operations happening before this could in the future influence the address. Hence, it is best ti print as late as possible.